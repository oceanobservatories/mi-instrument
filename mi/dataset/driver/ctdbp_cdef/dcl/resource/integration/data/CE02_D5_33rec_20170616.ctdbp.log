2017/06/16 00:00:04.001 [ctdbp:DLOGP3]:€SBE 16plus
2017/06/16 00:00:07.430 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/06/16 00:00:20.946 # 12.9489,  3.61935,    7.748, 16 Jun 2017 00:00:13
2017/06/16 00:00:30.940 # 12.9446,  3.61900,    6.182, 16 Jun 2017 00:00:23
2017/06/16 00:00:40.940 # 12.9420,  3.61896,    6.370, 16 Jun 2017 00:00:33
2017/06/16 00:00:50.954 # 12.9452,  3.61905,    5.700, 16 Jun 2017 00:00:43
2017/06/16 00:01:00.940 # 12.9313,  3.61862,    6.101, 16 Jun 2017 00:00:53
2017/06/16 00:01:10.940 # 12.9460,  3.61910,    6.667, 16 Jun 2017 00:01:03
2017/06/16 00:01:20.940 # 12.9379,  3.61887,    7.646, 16 Jun 2017 00:01:13
2017/06/16 00:01:30.940 # 12.9540,  3.61941,    6.745, 16 Jun 2017 00:01:23
2017/06/16 00:01:40.940 # 12.9329,  3.61850,    6.657, 16 Jun 2017 00:01:33
2017/06/16 00:01:50.940 # 12.9370,  3.61865,    6.654, 16 Jun 2017 00:01:43
2017/06/16 00:02:00.941 # 12.9372,  3.61891,    6.797, 16 Jun 2017 00:01:53
2017/06/16 00:02:10.945 # 12.9557,  3.61958,    4.638, 16 Jun 2017 00:02:03
2017/06/16 00:02:20.941 # 12.9561,  3.61963,    6.685, 16 Jun 2017 00:02:13
2017/06/16 00:02:30.952 # 12.9568,  3.61975,    7.282, 16 Jun 2017 00:02:23
2017/06/16 00:02:40.943 # 12.9528,  3.61953,    6.968, 16 Jun 2017 00:02:33
2017/06/16 00:02:50.948 # 12.9523,  3.61954,    7.130, 16 Jun 2017 00:02:43
2017/06/16 00:03:07.425 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/06/16 14:00:04.499 [ctdbp:DLOGP3]:€SBE 16plus
2017/06/16 14:00:07.917 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/06/16 14:00:20.962 # 13.0637,  3.61873,    7.289, 16 Jun 2017 14:00:13
2017/06/16 14:00:30.957 # 13.0644,  3.61882,    7.198, 16 Jun 2017 14:00:23
2017/06/16 14:00:40.957 # 13.0622,  3.61855,    7.167, 16 Jun 2017 14:00:33
2017/06/16 14:00:50.957 # 13.0631,  3.61865,    6.964, 16 Jun 2017 14:00:43
2017/06/16 14:01:00.958 # 13.0629,  3.61868,    7.442, 16 Jun 2017 14:00:53
2017/06/16 14:01:10.956 # 13.0628,  3.61871,    7.269, 16 Jun 2017 14:01:03
2017/06/16 14:01:20.957 # 13.0661,  3.61854,    7.173, 16 Jun 2017 14:01:13
2017/06/16 14:01:30.967 # 13.0678,  3.61910,    7.592, 16 Jun 2017 14:01:23
2017/06/16 14:01:40.957 # 13.0628,  3.61862,    7.685, 16 Jun 2017 14:01:33
2017/06/16 14:01:50.957 # 13.0573,  3.61806,    7.136, 16 Jun 2017 14:01:43
2017/06/16 14:02:00.956 # 13.0505,  3.61733,    6.690, 16 Jun 2017 14:01:53
2017/06/16 14:02:10.956 # 13.0579,  3.61802,    6.426, 16 Jun 2017 14:02:03
2017/06/16 14:02:20.956 # 13.0586,  3.61813,    7.001, 16 Jun 2017 14:02:13
2017/06/16 14:02:30.956 # 13.0591,  3.61814,    6.857, 16 Jun 2017 14:02:23
2017/06/16 14:02:40.957 # 13.0560,  3.61789,    7.106, 16 Jun 2017 14:02:33
2017/06/16 14:02:50.957 # 13.0561,  3.61782,    7.611, 16 Jun 2017 14:02:43
2017/06/16 14:03:00.957 # 13.0493,  3.61694,    7.351, 16 Jun 2017 14:02:53
2017/06/16 14:03:07.448 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
