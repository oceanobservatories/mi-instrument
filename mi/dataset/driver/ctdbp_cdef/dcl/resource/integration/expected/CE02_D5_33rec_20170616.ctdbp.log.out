
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560020.9460000992 |          5 |         3706560013 | 3706560020.9460000992 |      port_timestamp |           ok |      3.61935 |             null |                     null |    7.748 | 12.9489
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560030.9400000572 |          5 |         3706560023 | 3706560030.9400000572 |      port_timestamp |           ok |        3.619 |             null |                     null |    6.182 | 12.9446
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560040.9400000572 |          5 |         3706560033 | 3706560040.9400000572 |      port_timestamp |           ok |      3.61896 |             null |                     null |     6.37 |  12.942
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560050.9539999962 |          5 |         3706560043 | 3706560050.9539999962 |      port_timestamp |           ok |      3.61905 |             null |                     null |      5.7 | 12.9452
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560060.9400000572 |          5 |         3706560053 | 3706560060.9400000572 |      port_timestamp |           ok |      3.61862 |             null |                     null |    6.101 | 12.9313
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560070.9400000572 |          5 |         3706560063 | 3706560070.9400000572 |      port_timestamp |           ok |       3.6191 |             null |                     null |    6.667 |  12.946
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560080.9400000572 |          5 |         3706560073 | 3706560080.9400000572 |      port_timestamp |           ok |      3.61887 |             null |                     null |    7.646 | 12.9379
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560090.9400000572 |          5 |         3706560083 | 3706560090.9400000572 |      port_timestamp |           ok |      3.61941 |             null |                     null |    6.745 |  12.954
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560100.9400000572 |          5 |         3706560093 | 3706560100.9400000572 |      port_timestamp |           ok |       3.6185 |             null |                     null |    6.657 | 12.9329
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560110.9400000572 |          5 |         3706560103 | 3706560110.9400000572 |      port_timestamp |           ok |      3.61865 |             null |                     null |    6.654 |  12.937
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560120.9409999847 |          5 |         3706560113 | 3706560120.9409999847 |      port_timestamp |           ok |      3.61891 |             null |                     null |    6.797 | 12.9372
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560130.9449996948 |          5 |         3706560123 | 3706560130.9449996948 |      port_timestamp |           ok |      3.61958 |             null |                     null |    4.638 | 12.9557
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560140.9409999847 |          5 |         3706560133 | 3706560140.9409999847 |      port_timestamp |           ok |      3.61963 |             null |                     null |    6.685 | 12.9561
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560150.9519996643 |          5 |         3706560143 | 3706560150.9519996643 |      port_timestamp |           ok |      3.61975 |             null |                     null |    7.282 | 12.9568
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560160.9429998398 |          5 |         3706560153 | 3706560160.9429998398 |      port_timestamp |           ok |      3.61953 |             null |                     null |    6.968 | 12.9528
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706560170.9479999542 |          5 |         3706560163 | 3706560170.9479999542 |      port_timestamp |           ok |      3.61954 |             null |                     null |     7.13 | 12.9523
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610420.9619998932 |          5 |         3706610413 | 3706610420.9619998932 |      port_timestamp |           ok |      3.61873 |             null |                     null |    7.289 | 13.0637
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610430.9569997787 |          5 |         3706610423 | 3706610430.9569997787 |      port_timestamp |           ok |      3.61882 |             null |                     null |    7.198 | 13.0644
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610440.9569997787 |          5 |         3706610433 | 3706610440.9569997787 |      port_timestamp |           ok |      3.61855 |             null |                     null |    7.167 | 13.0622
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610450.9569997787 |          5 |         3706610443 | 3706610450.9569997787 |      port_timestamp |           ok |      3.61865 |             null |                     null |    6.964 | 13.0631
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610460.9580001831 |          5 |         3706610453 | 3706610460.9580001831 |      port_timestamp |           ok |      3.61868 |             null |                     null |    7.442 | 13.0629
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610470.9560003281 |          5 |         3706610463 | 3706610470.9560003281 |      port_timestamp |           ok |      3.61871 |             null |                     null |    7.269 | 13.0628
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610480.9569997787 |          5 |         3706610473 | 3706610480.9569997787 |      port_timestamp |           ok |      3.61854 |             null |                     null |    7.173 | 13.0661
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610490.9670000076 |          5 |         3706610483 | 3706610490.9670000076 |      port_timestamp |           ok |       3.6191 |             null |                     null |    7.592 | 13.0678
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610500.9569997787 |          5 |         3706610493 | 3706610500.9569997787 |      port_timestamp |           ok |      3.61862 |             null |                     null |    7.685 | 13.0628
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610510.9569997787 |          5 |         3706610503 | 3706610510.9569997787 |      port_timestamp |           ok |      3.61806 |             null |                     null |    7.136 | 13.0573
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610520.9560003281 |          5 |         3706610513 | 3706610520.9560003281 |      port_timestamp |           ok |      3.61733 |             null |                     null |     6.69 | 13.0505
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610530.9560003281 |          5 |         3706610523 | 3706610530.9560003281 |      port_timestamp |           ok |      3.61802 |             null |                     null |    6.426 | 13.0579
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610540.9560003281 |          5 |         3706610533 | 3706610540.9560003281 |      port_timestamp |           ok |      3.61813 |             null |                     null |    7.001 | 13.0586
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610550.9560003281 |          5 |         3706610543 | 3706610550.9560003281 |      port_timestamp |           ok |      3.61814 |             null |                     null |    6.857 | 13.0591
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610560.9569997787 |          5 |         3706610553 | 3706610560.9569997787 |      port_timestamp |           ok |      3.61789 |             null |                     null |    7.106 |  13.056
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610570.9569997787 |          5 |         3706610563 | 3706610570.9569997787 |      port_timestamp |           ok |      3.61782 |             null |                     null |    7.611 | 13.0561
 CE02SHSM | RID27 | 03-CTDBPC000 | 3706560000 | telemetered | 3706610580.9569997787 |          5 |         3706610573 | 3706610580.9569997787 |      port_timestamp |           ok |      3.61694 |             null |                     null |    7.351 | 13.0493

(33 rows)
